src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -210,7 +257,11 @@ const styles = StyleSheet.create({
     height: dimen.size.TOPIC_CURRENT_ITEM_HEIGHT,
     width: FULL_WIDTH,
     marginBottom: normalizeFontSizeByWidth(4),
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.white,
+    shadowColor: COLORS.black000,
+    shadowOffset: {width: 0, height: 4},
+    shadowOpacity: 0.3,
+    shadowRadius: 4
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `shadowColor`, `shadowOffset`, `shadowOpacity`, and `shadowRadius` properties have been added to the `cardContainer` style. This should add a shadow effect to the card container.
```diff
+    backgroundColor: COLORS.white,
+    shadowColor: COLORS.black000,
+    shadowOffset: {width: 0, height: 4},
+    shadowOpacity: 0.3,
+    shadowRadius: 4
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->